src/promptflow/promptflow/_sdk/_pf_client.py

```diff
@@ -46,33 +46,49 @@ def run(
         **kwargs,
     ) -> Run:
         """Run flow against provided data or run.
-        Note: at least one of data or run must be provided.
-
-        :param flow: path to flow directory to run evaluation
-        :param data: pointer to test data (of variant bulk runs) for eval runs
-        :param run:
-            flow run id or flow run
-            keep lineage between current run and variant runs
-            batch outputs can be referenced as ${run.outputs.col_name} in inputs_mapping
-        :param column_mapping: define a data flow logic to map input data, support:
-            from data: data.col1:
-            from run:
-                run.inputs.col1: if need reference run's inputs
-                run.output.col1: if need reference run's outputs
-            Example:
-                {"ground_truth": "${data.answer}", "prediction": "${run.outputs.answer}"}
-        :param variant: Node & variant name in format of ${node_name.variant_name}, will use default variant
-            if not specified.
-        :param connections: Overwrite node level connections with provided value.
-            Example: {"node1": {"connection": "new_connection", "deployment_name": "gpt-35-turbo"}}
+
+        .. note::
+            At least one of the ``data`` or ``run`` parameters must be provided.
+
+        .. admonition:: column_mapping
```

Comment:
looks like this is not well handled?

Reply:
![Uploading image.png…]()

Reply:
The image is broken, this is how my local build is:
![image](https://github.com/microsoft/promptflow/assets/2572521/8d5ade1c-d726-4546-9d01-5209d650c988)

Reply:
Yes. I was using wrong conda env. 

We might add some descriptive words here, before we introduce from data and from run

Reply:
Added some description.